### PR TITLE
Fix GithubProvider scope settings

### PR DESCRIPTION
**Pull request by schoenwaldnils**

### CODE REVIEW

**Review by balazsorban44 — CHANGES_REQUESTED** [review]
This setting can be dropped. The default scope is the same as defined here.

**Comment by schoenwaldnils** [comment]
Since this is an example project I'd prefer to keep it in here (maybe with another scope?) or take it out and have the documentation for the provider updated.
I had to search through other issues to find the right way to change the scope. (Also because the type definition of the provider options are not clear)

**Comment by balazsorban44** [comment]
For that, please refer to the documentation

https://next-auth.js.org/providers/github

https://next-auth.js.org/configuration/providers/oauth#authorization-option

File: pages/api/auth/[...nextauth].js [hunk 1]
```diff
@@ -37,7 +37,11 @@ export default NextAuth({
       clientId: process.env.GITHUB_ID,
       clientSecret: process.env.GITHUB_SECRET,
       // https://docs.github.com/en/developers/apps/building-oauth-apps/scopes-for-oauth-apps
-      scope: "read:user",
+      authorization: {
+        params: {
+          scope: "read:user",
+        },
+      },
```
Comment (balazsorban44) [inline pages/api/auth/[...nextauth].js:44]:
```suggestion
```

**Review by balazsorban44 — CHANGES_REQUESTED** [review]

**Pull request merged**